source/extensions/transport_sockets/tls/ssl_socket.cc

```diff
@@ -224,7 +224,10 @@ void SslSocket::drainErrorQueue() {
                                         absl::NullSafeStringView(ERR_reason_error_string(err))));
   }
   if (!failure_reason_.empty()) {
-    ENVOY_CONN_LOG(debug, "{}", callbacks_->connection(), failure_reason_);
+    ENVOY_CONN_LOG(
+        debug, "remote address:{},  {}", callbacks_->connection(),
```

Comment:
nit, the order is reversed。

Reply:
@daixiang0 , the log is as follows:
[2022-05-16 01:35:40.733][13][debug][connection] [source/extensions/transport_sockets/tls/ssl_socket.cc:230] [C2] Remote address:10.226.193.124,  TLS error: 268435696:SSL routines:OPENSSL_internal:UNSUPPORTED_PROTOCOL.
How do you think of the order?

Reply:
I thought the log maybe like `remote address: 10.22.33.44, [something else]`, the above is ok to me.

Small nit: remove extra blank like `"remote address: {}, {}"`

Reply:
fixed